src/CMakeLists.txt

```diff
@@ -43,10 +43,16 @@ else()
     endif()
 endif()
 
-if (NOT BLT_CXX_STD)
-    set(BLT_CXX_STD "c++11" CACHE STRING "")
+if (“${PROJECT_SOURCE_DIR}” STREQUAL “${CMAKE_SOURCE_DIR}”)
```

Comment:
Are those double-quote `"` characters, or MS Word style open-double-quote and close-double-quote characters?  Is GitHub messing with me?

Edit: Compare to (new) line 50.  Also check all the other comparisons of PROJECT_SOURCE_DIR with CMAKE_SOURCE_DIR.

Reply:
Thanks for pointing these out @agcapps.

I saw it too, but for some reason thought it was just how github was rendering them.

Reply:
its definitely a different character... i can copy and paste it to another screen and type double-quote and they are different. so weird.

Reply:
Chris, are you writing code in MS Word again?

Reply:
you caught me.... my secret shame. (it was a copy and paste from readthedocs)